problems/sumDigits.yaml

```diff
@@ -44,4 +44,5 @@ solution: |
     }
     return total;
   }
-timeout: 2500
\ No newline at end of file
+timeout: 2500
+userSubmitted: false
```

Comment:
Defining `userSubmitted` seems redundant to me as users would not have access to the codebase.

Rather than having to define this in all YAML files, I think we might just want to set any problems generated from YAML files as `userSubmitted: false` by default.

Reply:
good idea, thank you!!